packages/components/src/VirtualizedList/CellTitle/CellTitle.scss

```diff
@@ -47,38 +47,75 @@ $tc-list-title-icon-size: $svg-lg-size !default;
 }
 
 // manage actions display on row hover
-:global(.tc-list-large-row):hover :global(.tc-list-title .tc-actions),
-:global(.ReactVirtualized__Table__row):hover :global(.tc-list-title .tc-actions) {
-	opacity: 1;
-	transition: opacity 0.15s ease-in;
-	visibility: visible;
+:global(.tc-list-large-row),
+:global(.ReactVirtualized__Table__row) {
+	&:hover,
+	&:focus,
+	&:focus-within {
```

Comment:
this feature doesn't work on IE/Edge, do you have a workaround to handle that on this browser ?

Reply:
It's not worth! :)